### ISSUE TITLE
NVDA not reading search suggestion for recent files in Word

### ISSUE THREAD

**Issue by SMTestGH** [nvaccess/nvda#12962]
### Steps to reproduce:
Enable NVDA and open Word (any Word 365 build)
Open File > Open > Recent (Alt>F>O>R)
Tab to the recent files search box
Type the letter A
Search suggestions appear in a vertical list
Arrow up/down through the list

### Actual behavior:
NVDA does not read the listitems so users aren't able to read suggestions.

### Expected behavior:
as a user arrows through the list, NVDA should read the listitems

#### NVDA installed/portable/running from source:
Installed and Portable both reproduce this problem.

#### NVDA version:
2021.1

#### Windows version:
Windows 11

#### Name and version of other software in use when reproducing the issue:
Word, Office 365 version 2108 and newer 

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
N/A

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

NVDA log info:
Speaking [LangChangeCommand ('en_US'), 'Search', 'edit', 'Search your recent files by file name', CancellableSpeech (still valid), 't']
IO - inputCore.InputManager.executeGesture (12:18:50.362) - winInputHook (40580):
Input: kb(laptop):downArrow
IO - inputCore.InputManager.executeGesture (12:18:51.783) - winInputHook (40580):
Input: kb(laptop):upArrow
DEBUG - editableText.EditableText._hasCaretMoved (12:18:51.893) - MainThread (48400):
Caret didn't move before timeout. Elapsed: 100 ms
DEBUG - editableText.EditableText._hasCaretMoved (12:18:53.599) - MainThread (48400):
Caret didn't move before timeout. Elapsed: 100 ms
IO - speech.speech.speak (12:18:53.643) - MainThread (48400):
Speaking [LangChangeCommand ('en_US'), 't']

**Comment by Adriani90** [comment]
This works as expected in Microsoft Office 365 MSO (Version 2410 Build 16.0.18129.20158) 64 Bit
and NVDA 2024.4.1 or last alpha.

Closing as works for me.
